pallets/order-book/src/traits.rs

```diff
@@ -123,12 +123,21 @@ where
         order_book_id: &OrderBookId<AssetIdOf<T>, T::DEXId>,
     ) -> MarketSide<T::MaxSidePriceCount>;
 
-    /// Returns order ids of user
+    /// Returns order ids of user from the order book with `order_book_id`
     fn get_user_limit_orders(
         &mut self,
         account: &T::AccountId,
         order_book_id: &OrderBookId<AssetIdOf<T>, T::DEXId>,
     ) -> Option<UserOrders<T::OrderId, T::MaxOpenedLimitOrdersPerUser>>;
+
+    /// Returns order ids of user from all order books
+    fn get_all_user_limit_orders(
```

Comment:
is this method going to be used later? just want to clarify that it's needed

Reply:
It was used for previos solution until we decided to change the approach. It could be useful for some cases and I decided to not remove it